common.js

```diff
@@ -69,7 +69,8 @@ function generateFinalBasename (opts) {
 }
 
 function generateFinalPath (opts) {
-  return path.join(opts.out || process.cwd(), generateFinalBasename(opts))
+  if (opts.out) return path.join(opts.out, generateFinalBasename(opts))
```

Comment:
It would probably be better if it printed a warning in the case where `typeof(opts.out) not in ['undefined', 'string']` (to use pseudocode).